apps/core/scenes/search/SearchModal.js

```diff
@@ -65,6 +70,20 @@ class SearchModal extends React.Component<Props> {
             onSavePress={this.handlePassengersSave}
           />
         )}
+        {[MODAL_TYPE.ORIGIN, MODAL_TYPE.DESTINATION].includes(
+          this.props.modalType,
+        ) && (
+          <View
+            style={{
+              margin: 25,
+              flexDirection: 'row',
+              alignItems: 'center',
+            }}
```

Comment:
Please create a style with `StyleSheet.create` instead

Reply:
this is just temporary so I will change it in next PR

Reply:
ok